crates/nu-command/src/formats/to/nuon.rs

```diff
@@ -1,4 +1,6 @@
 use core::fmt::Write;
+use fancy_regex::Regex;
+use lazy_static::lazy_static;
```

Comment:
Going forward, we generally want to use `once_cell` instead of `lazy_static`. `once_cell` is probably going to be in the stdlib eventually and I believe the ecosystem has started to standardize on it accordingly: https://github.com/rust-lang/rust/issues/74465